### ISSUE TITLE
make bootstrap-client produces multiple fail messages

### ISSUE THREAD

**Issue by Mozahler** [pointfreeco/isowords#9]
I'm new to git-lfs and the version bootstrap tried to use caused a segmentation fault. I removed it and reinstalled using brew.
Then I was asked to issue 'git lfs install' when I issued 'git-lfs' with no args. That worked fine.
At this point I am cd-ed into isowords, and issue 'make bootstrap-client':

  ⚠️  Checking for Git LFS...
  ✅ Git LFS is good to go!
touch Sources/AppAudioLibrary/Resources/empty.mp3
touch Sources/AppClipAudioLibrary/Resources/empty.mp3
/dev/stdin:173: INSERT failed: UNIQUE constraint failed: words.word
/dev/stdin:236: INSERT failed: UNIQUE constraint failed: words.word
--- for a total of 1468 similar lines

are these normal error messages that can be safely ignored?

**Comment by Mozahler** [comment]
I continued with the installation, running 'make bootstrap' which told me I needed to install and then start the PostgreSQL db.
After doing that, I ran 'make bootstrap' again. This is what happened:

 ⚠️  Checking on PostgreSQL...
  ✅ PostgreSQL is up and running!
2021-03-18 09:54:48.369 EDT [7496] FATAL:  role "isowords" does not exist
  ⚠️  Local databases aren't configured! Creating isowords user/databases...

     Reset at any time with:

       $ make db-drop

ALTER ROLE
2021-03-18 09:54:48.629 EDT [7512] ERROR:  role "isowords" already exists
2021-03-18 09:54:48.629 EDT [7512] STATEMENT:  CREATE ROLE isowords SUPERUSER CREATEDB CREATEROLE INHERIT LOGIN;
createuser: error: creation of new role failed: ERROR:  role "isowords" already exists
ALTER ROLE
2021-03-18 09:54:48.659 EDT [7517] ERROR:  database "isowords_development" already exists
2021-03-18 09:54:48.659 EDT [7517] STATEMENT:  CREATE DATABASE isowords_development OWNER isowords;
createdb: error: database creation failed: ERROR:  database "isowords_development" already exists
2021-03-18 09:54:48.676 EDT [7520] ERROR:  database "isowords_test" already exists
2021-03-18 09:54:48.676 EDT [7520] STATEMENT:  CREATE DATABASE isowords_test OWNER isowords;
createdb: error: database creation failed: ERROR:  database "isowords_test" already exists

Am I good to go?

I'll try to build and run and post any updates here.
